server/pulp/server/webservices/views/consumer_groups.py

```diff
@@ -16,6 +18,17 @@
                                                 json_body_required)
 
 
+def serialize_consumer_group(group):
```

Comment:
Nitpick, but you could just name this serialize() since it's in a module called consumer_groups.py. Then, for modules that import consumer_groups, they can call it with consumer_groups.serialize() which reads nicely.